source/utils/requestAnimationTimeout.js

```diff
@@ -19,7 +19,11 @@ export const requestAnimationTimeout = (
   callback: Function,
   delay: number,
 ): AnimationTimeoutId => {
-  const start = Date.now();
+  let start;
+  // wait for end of processing current event handler, because event handler may be long
+  Promise.resolve().then(() => {
```

Comment:
FYI this adds core-js polyfill to the output

Reply:
I have no idea what to do with it.
Looks like we'll have to leave it at that.